contracts/test/integration/IntegrationTestPriceBoundPSMDai.t.sol

```diff
@@ -0,0 +1,340 @@
+import {getCore, getMainnetAddresses, VoltTestAddresses} from "../unit/utils/Fixtures.sol";
+import {ERC20CompoundPCVDeposit} from "../../pcv/compound/ERC20CompoundPCVDeposit.sol";
+import {Vm} from "./../unit/utils/Vm.sol";
+import {DSTest} from "./../unit/utils/DSTest.sol";
+import {MainnetAddresses} from "./fixtures/MainnetAddresses.sol";
+import {Constants} from "../../Constants.sol";
+
+contract IntegrationTestPriceBoundPSMDaiTest is DSTest {
+    using SafeCast for *;
+    PriceBoundPSM private psm;
+    ICore private core = ICore(MainnetAddresses.CORE);
+    IVolt private volt = IVolt(MainnetAddresses.VOLT);
+    IVolt private dai = IVolt(MainnetAddresses.DAI);
+    IVolt private underlyingToken = dai;
+
+    MockPCVDepositV2 public pcvDeposit;
+
+    /// ------------ Minting and RateLimited System Params ------------
+
+    uint256 public constant mintAmount = 10_000_000e18;
+    uint256 public constant mintAmountPSM = 10_000_000e18;
+    uint256 public constant bufferCap = 10_000_000e18;
+    uint256 public constant individualMaxBufferCap = 5_000_000e18;
+    uint256 public constant rps = 10_000e18;
+
+    /// ------------ Oracle System Params ------------
+
+    /// @notice prices during test will increase 1% monthly
+    int256 public constant monthlyChangeRateBasisPoints = 100;
+    uint256 public constant maxDeviationThresholdBasisPoints = 1_000;
+
+    /// @notice Oracle Pass Through contract
+    OraclePassThrough public oracle =
+        OraclePassThrough(MainnetAddresses.ORACLE_PASS_THROUGH);
+
+    Vm public constant vm = Vm(HEVM_ADDRESS);
+
+    uint256 voltFloorPrice = 9_000; /// 1 volt for .9 dai is the max allowable price
+    uint256 voltCeilingPrice = 10_000; /// 1 volt for 1 dai is the minimum price
+
+    function setUp() public {
+        PegStabilityModule.OracleParams memory oracleParams;
+
+        pcvDeposit = new MockPCVDepositV2(
+            address(core),
+            address(underlyingToken),
+            0,
+            0
+        );
+
+        oracleParams = PegStabilityModule.OracleParams({
+            coreAddress: address(core),
+            oracleAddress: address(oracle),
+            backupOracle: address(0),
+            decimalsNormalizer: 0,
+            doInvert: true
+        });
+
+        /// create PSM
+        psm = new PriceBoundPSM(
+            voltFloorPrice,
+            voltCeilingPrice,
+            oracleParams,
+            0,
+            0,
+            10_000_000_000e18,
+            10_000e18,
+            10_000_000e18,
+            IERC20(address(dai)),
+            pcvDeposit
+        );
+
+        vm.startPrank(MainnetAddresses.DAI_USDC_USDT_CURVE_POOL);
+        dai.transfer(address(this), mintAmount);
+        dai.transfer(address(psm), mintAmount * 2);
+        vm.stopPrank();
+
+        vm.startPrank(MainnetAddresses.GOVERNOR);
+
+        /// grant the PSM the PCV Controller role
+        core.grantMinter(MainnetAddresses.GOVERNOR);
```

Comment:
you could just impersonate the governor and transfer VOLT because on mainnet the msig has a whole lot of VOLT

Reply:
your way is better because then the tests won't break if those VOLT get transferred.